app/controllers/readings_controller.rb

```diff
@@ -0,0 +1,25 @@
+class ReadingsController < ApplicationController
+  def index
+    @readings = Reading.all
```

Comment:
Perhaps would make sense to scope readings to it's sensor, rather than showing all the readings from all the sensors